.github/workflows/docker-image-push.yml

```diff
@@ -0,0 +1,58 @@
+name: Build (and push) multi-arch Docker image
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    # We want a newer version of qemu: https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1815100
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile: [Dockerfile, alpine/Dockerfile]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Docker login
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        run: echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin docker.io
+
+      - name: Setup buildx
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu-user-static
+          docker buildx create --use
+
+      - name: Build Docker image
+        env:
+          PLATFORMS: linux/amd64,linux/arm64/v8,linux/arm/v7
+          DOCKER_REPOSITORY: thelounge/thelounge
+        run: |
+          TAG="$(git rev-parse --short HEAD)"
```

Comment:
is `git rev-parse --short HEAD` really the correct way to get the tag name (e.g. 4.2.0-pre.2) at HEAD?

Reply:
It isn't, but it is only used on regular push. When a tag is pushed `VERSION="$(git describe --exact-match --tags)"` is used (line 41).